skiko/src/awtMain/cpp/windows/direct3DContext.cc

```diff
@@ -18,10 +18,8 @@ extern "C"
         {
             SkSurface *surface = fromJavaPointer<SkSurface *>(surfacePtr);
             GrDirectContext *context = fromJavaPointer<GrDirectContext *>(contextPtr);
-            surface->flushAndSubmit(true);
-            surface->flush(SkSurface::BackendSurfaceAccess::kPresent, GrFlushInfo());
-            context->flush();
-            context->submit(true);
+            context->flush(surface, SkSurfaces::BackendSurfaceAccess::kPresent, GrFlushInfo());
```

Comment:
@igordmn Could you please take a look at just this particular change for now?
SkSurface doesn't have flushAndSubmit, flush, submit methods anymore: https://github.com/google/skia/blob/main/RELEASE_NOTES.md#milestone-116

> SkSurface::flushAndSubmit -> GrDirectContext::flushAndSubmit
SkSurface::flush -> GrDirectContext::flush

I guess the API change just simplified the usage and ensure a correct sequence of the calls, but I'm not familiar with the responsibilities of the involved parts and methods.

Reply:
I don't know details here, but looks legit. The tests will show if it works correctly.